pyaci/core.py

```diff
@@ -266,9 +266,16 @@ def startWsListener(self):
             try:
                 proxyUrl = self._userProxies.get("https", self._userProxies.get("http", None))
                 if proxyUrl:
+                    # proxy format: http://username:password@host:port
+                    # if username:password is provided, it has to be removed in
+                    # order to retrive host and port properly.
+                    if "@" in proxyUrl:
+                        proxyUrl = "http://{}".format(proxyUrl.split("@")[1])
```

Comment:
Should we handle `https` correctly? This looks like we'll be forcing `http` irrespective of what is set in the `_userProxies`.